apps/web/src/features/enrollments/components/enrollment.detail.skeleton.tsx

```
@@ -10,36 +10,36 @@ export default function EnrollmentDetailSkeleton({
 					</div>
 				</div>
 			</div>
 
 			<div className="space-y-4 px-6 py-4">
 				{/* Cards skeleton */}
 				<div className="rounded-lg border bg-background p-6">
-					<div className="mb-4 h-6 w-40 rounded bg-muted animate-pulse" />
+					<div className="mb-4 h-6 w-40 animate-pulse rounded bg-muted" />
 					<div className="grid gap-8 lg:grid-cols-3">
 						{[...Array(3)].map((_, i) => (
 							<div key={i} className="space-y-4">
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)